mm2src/trezor/src/eth/eth_command.rs

```diff
@@ -0,0 +1,214 @@
+use crate::proto::{messages_ethereum as proto_ethereum, messages_ethereum_definitions as proto_ethereum_definitions};
+use crate::response_processor::ProcessTrezorResponse;
+use crate::result_handler::ResultHandler;
+use crate::{serialize_derivation_path, OperationFailure, TrezorError, TrezorResponse, TrezorResult, TrezorSession};
+use ethcore_transaction::{signature, Action, Transaction as UnSignedEthTx, UnverifiedTransaction as UnverifiedEthTx};
+use ethereum_types::H256;
+use ethkey::Signature;
+use hw_common::primitives::{DerivationPath, XPub};
+use lazy_static::lazy_static;
+use mm2_err_handle::map_mm_error::MapMmError;
+use mm2_err_handle::or_mm_error::OrMmError;
+use mm2_err_handle::prelude::MmError;
+use std::collections::BTreeMap;
+
+type ChainId = u64;
+type StaticDefinitionBytes = &'static [u8];
+type StaticAddressBytes = &'static [u8];
+
+// new supported eth networks:
+const SEPOLIA_ID: u64 = 11155111;
+
+lazy_static! {
+
+    // External eth network definitions
+    static ref ETH_NETWORK_DEFS: BTreeMap<ChainId, StaticDefinitionBytes> = [
+        (SEPOLIA_ID, SEPOLIA_NETWORK_DEF.as_ref())
+    ].iter().cloned().collect();
+
+    // External eth token definitions
+    static ref ETH_TOKEN_DEFS: BTreeMap<StaticAddressBytes, (ChainId, StaticDefinitionBytes)> = [
+    ].iter().cloned().collect();
+
+    static ref SEPOLIA_NETWORK_DEF: Vec<u8> = include_bytes!("definitions/sepolia.dat").to_vec();
+    // add more files with external network or token definitions
+}
+
+/// Get external network definition by chain id
+/// check this doc how to find network definition files https://docs.trezor.io/trezor-firmware/common/ethereum-definitions.html
+fn get_eth_network_def(chain_id: ChainId) -> Option<Vec<u8>> {
+    ETH_NETWORK_DEFS
+        .iter()
+        .find(|(id, _def)| id == &&chain_id)
+        .map(|found| found.1.to_vec())
+}
+
+/// Get external token definition by token contract address and chain id
+/// check this doc how to find token definition files https://docs.trezor.io/trezor-firmware/common/ethereum-definitions.html
+#[allow(dead_code)]
+fn get_eth_token_def(address_bytes: &[u8], chain_id: ChainId) -> Option<Vec<u8>> {
+    ETH_TOKEN_DEFS
+        .iter()
+        .find(|(address, def)| address == &&address_bytes && def.0 == chain_id)
+        .map(|found| found.1 .1.to_vec())
+}
+
+/// trim leading zeros in array
+macro_rules! trim_left {
+    ($param:expr) => {{
+        $param.iter().skip_while(|el| el == &&0).cloned().collect::<Vec<_>>()
+    }};
+}
+
+impl<'a> TrezorSession<'a> {
+    /// Unused for now. Added for future use like implementation of an rpc returning eth address
```

Comment:
Plz fix this note that this fn is unused. This is not true now

Reply:
Fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/1979/commits/ee8ecf34248247a332045f4170d60766a2fe3e7f